addons/mail_bot/models/res_users.py

```diff
@@ -25,9 +25,13 @@ def SELF_READABLE_FIELDS(self):
         return super().SELF_READABLE_FIELDS + ['odoobot_state']
 
     def _init_messaging(self):
+        odoobot_onboarding = False
         if self.odoobot_state in [False, 'not_initialized'] and self._is_internal():
+            odoobot_onboarding = True
             self._init_odoobot()
-        return super()._init_messaging()
+        res = super()._init_messaging()
+        res['odoobotOnboarding'] = odoobot_onboarding
+        return res
```

Comment:
not sure which one I prefer, but here's an alternative way to do it though:
```suggestion
        return {
            **super()._init_messaging(),
            "odoobotOnboarding": odoobot_onboarding,
        }
```

Reply:
Pretty much same amount of LOCs, theoretically shallow copy less performant. I'm keeping previous version, though it's not a big deal